ros/src/computing/perception/localization/packages/kalman_filter_localizer/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kalman_filter_localizer</name>
+  <version>0.0.0</version>
+  <description>The kalman_filter package</description>
+
+
+  <maintainer email="takamasa.horibe@tier4.jp">horibe</maintainer>
+
+
+  <license>Appache2</license>
```

Comment:
```suggestion
  <license>Apache 2.0</license>
```

Reply:
I've changed following your correction. Thank you.